text/0118-search-api.md

```diff
@@ -374,6 +380,76 @@ is equivalent to:
 }
 ```
 
+###### 3.1.2.1.13 Is Empty
+
+The `IS EMPTY` operator selects the documents for which the filterable attribute exists and is empty. If the attribute doesn't exists then it is not empty and the document will not be returned. It is a postfix operator that takes an attribute name as argument.
+
+The negated form of `IS EMPTY` can be written in two ways:
+```
+attribute IS NOT EMPTY
+NOT attribute IS EMPTY
+```
+Both forms are equivalent. They select the documents for which the attribute is not empty.
```

Comment:
Could we add a bullet points list explaining what is considered empty for an attribute?

e.g

- `""`
- `[]`
- `{}`